test/FunctionalTests/Tests/DataServices/UnitTests/ClientCSharpUnitTests/DataWebClientCSharp/LinqTests.cs

```diff
@@ -8035,24 +8035,73 @@ orderby t.TeamID
             }
             catch (Exception e)
             {
-                if (e.Message != "Can't add query option '$custom' because it begins with reserved character '$'.")
+                if (e.Message != "The query option '$custom' is not supported.")
                 {
                     throw new Exception("Test Failed");
                 }
             }
 
+            var queryableApplyOption = ((DataServiceQuery<League>)from l in context.CreateQuery<League>("Leagues")

```

Comment:
Shouldnt $apply be supported?

Reply:
When you allow `$apply` to be added, a `PlatformNotSupported` exception is thrown with the error message "Secure binary serialization is not supported on this platform". I have not investigated this exception in depth.

Reply:
I don't think $apply is currently supported in Client